_posts/2021-06-18-r2diidata-020/r2diidata-020.Rmd

```diff
@@ -0,0 +1,72 @@
+---
+title: "r2dii.data 0.2.0 is now on CRAN"
+description: |
+  Signifcant changes to `ald_demo`
+author:
+  - name: Jackson Hoffart
+    url: https://github.com/jdhoffa
+base_url: https://2degreesinvesting.github.io/
+date: 2021-06-18
+output: 
+  distill::distill_article:
+    toc: true
+    toc_depth: 3
+    self_contained: true
+categories:
+  - r2dii
+  - software development
+  - release
+preview: preview.jpg
+twitter:
+  site: "@jdhoffa"
+  creator: "@jdhoffa"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+r2dii.data 0.2.0 is now on CRAN. This release fixes a number of bugs that you can learn about [here](https://2degreesinvesting.github.io/r2dii.data/news/index.html); this post shows enhancements and new features.
+
+You can install it from CRAN with:
+
+```r
+install.packages("r2dii.data")
+```
+
+Then use them with:
+
+```{r}
+library(r2dii.data)
+library(r2dii.match)
+```
+
+The most notable update in this release relates to changes in the demonstration dataset `ald_demo`. This data is meant to reflect a dataset that is provided to users by the organization [Asset Resolution](https://asset-resolution.com/).
+
+### `ald_demo` gains a column, loses a column an has two renamed
+
+`ald_demo` gains the new column `lei`, loses the column `is_ultimate_listed_owner` and has two columns renamed: 
+* `id_company` becomes `company_id`
+* `ald_emission_factor_unit` becomes `emission_factor_unit`
+
+The new column `lei` contains fake legal entity identifiers for some companies in `ald_demo`: 
+
+``` {r} 
+head(unique(ald_demo$lei))
+```
+
+Changes to the other columns are simply cosmetic, and meant to better reflect the actual dataset that will be delivered to users. 
+
+Since these fields are not currently used in the downstream packages `r2dii.match` and `r2dii.analysis`, it is not expected that this should negatively affect users in a significant way. 
+
```

Comment:
Maybe be a bit more personal: ... [we] don't expect this changes to affect [you] in any negative way.